src/swarm/neo/request/Command.d

```diff
@@ -61,6 +61,11 @@ public alias byte StatusCode;
 
 public enum GlobalStatusCode : StatusCode
```

Comment:
Maybe add `alias GlobalStatusCode SupportedStatus;`.

Reply:
Oh, or would renaming the enum and adding a deprecated alias do the job? (As long as we change all usages inside swarm, we shouldn't get deprecation warnings.)

Reply:
No, sadly, this is used outside the swarm (the node handlers are sending these codes to the clients).

Reply:
Right, that would mean that when we update *proto to use swarm v4.5.0, we'd get deprecation warnings that we'd then fix. Or?

Reply:
Right, that would work, it's just renaming in this case.